### DISCUSSION TITLE
Privado VPN: Currently no server in Switzerland

### DISCUSSION THREAD

**Discussion by moglifreitag** [qdm12/gluetun#2117] [Q&A] [answered]
This is not an issue caused by gluetun. Just want to share this information with the community. 

The hostnames for the Privado VPN server in Switzerland have changed twice recently. 
About a week ago they changed from zrh-001-4 to zrh-005-8. 
A few days ago they changed again, from zrh-005-8 to zrh-009-12. 

No problem, this can be easily fixed with updating the servers.json... BUT with the last change the current hostnames point to server, which are apparently not located in Switzerland. 
Based on gluetun servers are located in Germany (the reason why SERVER_COUNTRIES=Switzerland does not work).
RIPE database points to Netherlands, also a traceroute ends there. 
Be aware that using SERVER_HOSTNAMES=zrh-009.vpn.privado.io (or the other current zrh-0xx) will work, but you are not connected to a server in Switzerland. 

Past (at least for some month) and current hostnames and IP network for Privado VPN server in Switzerland: 
zrh-00(1-4).vpn.privado.io
--> 185.156.175.0 - 185.156.175.255
--> country:        CH

zrh-00(5-8).vpn.privado.io
--> 146.70.26.0 - 146.70.26.255
--> country:        CH

zrh-0(09-12).vpn.privado.io
--> 91.148.192.0 - 91.148.255.255 
--> country:        NL

**Comment by moglifreitag** [comment]
VPN server in Switzerland available again. Hostnames changed back to 
zrh-00(5-8).vpn.privado.io
--> 146.70.26.0 - 146.70.26.255
--> country: CH

**Comment by qdm12** [comment, marked as answer]
See https://github.com/qdm12/gluetun/issues/2118#issuecomment-2262936929
